src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -58,6 +58,13 @@
    :dependencies (schema/pred sequential?)
    :fns-map FnsMap})
 
+(def ProtocolandDependenciesMap
+  "Schema defining the map used to represent the protocol and dependencies
+  of a service."
+  {:fns [clojure.lang.PersistentList]
```

Comment:
@nathanielksmith I was chatting with @janelu2 and she said that you had a preference for the Java class here.  I'm not necessarily opposed, but am interested in hearing your reasoning.